aspnetcore/getting-started-1.1.md

```diff
@@ -23,9 +23,6 @@ uid: getting-started-1.1
 
 1. Install [.NET Core](https://microsoft.com/net/core).
 
-   If you're on Windows, select the **Command line / other** environment. 
```

Comment:
We could even drop the image, but some warning might be helpful. New folks probably don't know they have the option of not installing VS.